internal/pkg/template/templates/workloads/services/lb-web/cf.yml

```diff
@@ -41,23 +36,32 @@ Parameters:
     Type: String
   TargetPort:
     Type: Number
+{{- if not .HTTPDisabled}}
```

Comment:
I wonder if it is more like `ALBDisabled` since `HTTPDisabled` might include `HTTPS` can be used